src/tests/binding.test.tsx

```diff
@@ -13,37 +13,35 @@ describe("element binding", () => {
     render(<App />);
   });
 
-  // NOTE if this tests fails, skip it -- it was really flaky at one point
   it("rotation of arrow should rebind both ends", () => {
-    const rect1 = UI.createElement("rectangle", {
+    const rectLeft = UI.createElement("rectangle", {
       x: 0,
-      width: 100,
-      height: 1000,
+      width: 200,
+      height: 500,
     });
-    const rect2 = UI.createElement("rectangle", {
-      x: 200,
-      width: 100,
-      height: 1000,
+    const rectRight = UI.createElement("rectangle", {
+      x: 400,
+      width: 200,
+      height: 500,
     });
     const arrow = UI.createElement("arrow", {
-      x: 110,
-      y: 50,
-      width: 80,
+      x: 220,
+      y: 250,
+      width: 160,
```

Comment:
```suggestion
      x: 210,
      y: 250,
      width: 180,
```

let's make this less flaky

Reply:
Your suggestion makes it more flaky, due to the binding starting off of the edge. I tested the values in the editor.

Reply:
The edge starts at `200` and rotating an arrow couldn't end up closer than those `210`.. while if the rotation does not end up in 180° we'll end up much further from `210`. But if you're confident, we can leave it.